colossalai/gptq/cai_gptq/__init__.py

```diff
@@ -9,6 +9,6 @@
     HAS_AUTO_GPTQ = False
 
```

Comment:
(Just wonder) if we would like to keep the current dir structure (e.g. colossalai.gptq). Refer to Coati inference file structure, `Chat/coati/quant/llama_gptq/`

Reply:
how about colossalai/inference/quant/gptq?